src/components/Layout/Header/NavBar/MainNavLink.tsx

```diff
@@ -30,6 +31,7 @@ export const MainNavLink = memo(
       >
         <Box display={{ base: isCompact ? 'none' : 'flex', '2xl': 'block' }}>{label}</Box>
       </Button>
```

Comment:
Should be indented too, I think ESLint will throw it, with prettier

Reply:
I modified the linting with ESLint, i think it is correct now